satpy/Combined_colormaps.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>Thoughts on turning off warnings to "clean up" the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Em ... Do you have any idea of how to turn the warning off? just using <code>%%capture</code>  or any other method?</p>

Reply:
See: https://github.com/pytroll/tutorial-satpy-half-day/blob/adfa08c7458ebf09e7a8c0148118afcfb2176c95/init_notebook.py#L6-L7